ros2action/ros2action/verb/send_goal.py

```diff
@@ -51,6 +58,10 @@ def main(self, *, args):
         feedback_callback = None
         if args.feedback:
             feedback_callback = _feedback_callback
+
+        if args.stdin:
+            args.goal = correct_stdin()
+
         return send_goal(args.action_name, args.action_type, args.goal, feedback_callback)
```

Comment:
Looking at this again, I think we should do this slightly differently:

```suggestion
        if args.stdin:
            goal = collect_stdin()
        else:
            goal = args.goal

        return send_goal(args.action_name, args.action_type, goal, feedback_callback)
```

Reply:
Fixed.